scripts/partition_manager/partition_manager.rst

```diff
@@ -642,6 +642,42 @@ The Partition Manager stores all variables as target properties on the ``partiti
 
    --slot-size $<TARGET_PROPERTY:partition_manager,PM_MCUBOOT_PARTITIONS_PRIMARY_SIZE>
 
+.. _pm_partition_reports:
+
+Partition Placement Reports
+---------------------------
+
+You can generate ASCII-art map reports using the `ninja partition_manager_report` command to get an overview of how the partition manager creates flash memory partitions. This is especially useful when using multiple bootloaders.
```

Comment:
nRF doc styleguide advises to keep one sentence per line

Reply:
Good catch, fixed in latest commit.